sentinelhub/constants.py

```diff
@@ -160,6 +196,10 @@ def get_wfs_typename(cls, data_source):
         :rtype: str
         """
         is_eocloud = SHConfig().is_eocloud_ogc_url()
+
+        if 'BYOC_' in str(data_source):
+            return 'DSS10-{}'.format(str(data_source.value))
```

Comment:
Whenever you use `format` you don't have to cast arguments to a string because it is done automatically. Besides, `data_source.value` should already be a string.